### PR TITLE
Emil changed test script

### PR DESCRIPTION
**Pull request by LukasBHolst**

### CODE REVIEW

File: src/video_emotion/extract_test_data.py [hunk 1]
```diff
@@ -1,27 +1,92 @@
 import sys
+import os
+from os import listdir
+from os.path import isfile, join
 
-from video_emotion.extract_classifier import classify_video
+sys.path.insert(0, os.path.abspath("../"))
```
Comment (QuincySharp) [inline src/video_emotion/extract_test_data.py]: why no get absolute path library?

File: src/video_emotion/extract_classifier.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import sys
-
+import os
+sys.path.insert(0, os.path.abspath("../"))
```
Comment (QuincySharp) [inline src/video_emotion/extract_classifier.py]: why no get absolute path library?

File: src/video_emotion/extract_test_data.py [hunk 3]
```diff
@@ -1,27 +1,92 @@
...
+
+import cv2
+import numpy as np
 
+from video_emotion.extract_classifier import classify_video
+from video_emotion.facial_recognition.facial_recognition import analyze_video
+from video_emotion.emotionTagger.face_emotion_extraction import classify_faces
 
 def generate_log_data(interval, emotion_set):
     output_log_data = ""
```
Comment (QuincySharp) [inline src/video_emotion/extract_test_data.py]: Why is this line needed, instead of just the one below, without the '+'

File: src/video_emotion/facial_recognition/facial_recognition.py [hunk 4]
```diff
@@ -50,8 +50,7 @@ def analyze_video(video_path, time_range=None):
 
         # Add found frames to our dictionary
         if len(faces) > 0:
-            frame_number = int(cap.get(cv2.CAP_PROP_POS_FRAMES))
-            dict_of_faces[str(frame_number)] = faces
+            dict_of_faces[str(int(cap.get(cv2.CAP_PROP_POS_MSEC)))] = faces
```
Comment (QuincySharp) [inline src/video_emotion/facial_recognition/facial_recognition.py]: Maybe a comment to understand what is going on here. Might not be needed

File: src/video_emotion/t/test_extract_test_data.py [hunk 5]
```diff
@@ -1,6 +1,31 @@
...
+    assert data == ("From,To,"
+                    + "emotion_angry,emotion_disgust,"
+                    + "emotion_fear,emotion_happy,emotion_sad,"
+                    + "emotion_surprise,emotion_neutral\n"
+                    + "0,500,10,5,10,10,5,5,5\n")
+
+
+def test_generate_log_data_images_cutoff_exists():
+    test_filename = get_absolute_path("video_emotion/t/test_video_emotion/Fun_at_a_Fair.mp4")
+    get_absolute_path
```
Comment (QuincySharp) [inline src/video_emotion/t/test_extract_test_data.py]: What does this do, and why is it needed?

File: src/video_emotion/extract_classifier.py (hunk 2 above)
Comment (LukasBHolst) [reply src/video_emotion/extract_classifier.py]: We can't use it in this context

File: src/video_emotion/extract_test_data.py (hunk 1 above)
Comment (LukasBHolst) [reply src/video_emotion/extract_test_data.py]: We can't use it in this context

File: src/video_emotion/extract_test_data.py (hunk 3 above)
Comment (EmilHenriksen) [reply src/video_emotion/extract_test_data.py]: sure

File: src/video_emotion/facial_recognition/facial_recognition.py (hunk 4 above)
Comment (EmilHenriksen) [reply src/video_emotion/facial_recognition/facial_recognition.py]: done

File: src/video_emotion/extract_test_data.py (hunk 3 above)
Comment (LukasBHolst) [reply src/video_emotion/extract_test_data.py]: Resolved

**Comment by LukasBHolst** [comment]
Nice implementation
